src/libasr/pass/pass_list_expr.cpp

```diff
@@ -152,7 +152,7 @@ class ListExprReplacer : public ASR::BaseExprReplacer<ListExprReplacer>
         */
 
         SymbolTable* list_section_symtab = al.make_new<SymbolTable>(global_scope);
-        std::string list_type_name = ASRUtils::get_type_code(list_type, true);
+        std::string list_type_name = ASRUtils::type_to_str_python(list_type);
```

Comment:
I think this needs to be reverted. Please see https://github.com/lcompilers/lpython/commit/a166236d1ccc5a9d6c49c44f7e9751223de339f1.